packages/mason_logger/test/src/progress_test.dart

```diff
@@ -49,6 +49,31 @@ void main() {
       });
     });
 
+    group('.update', () {
+      test('writes lines to stdout', () async {
+        await StdioOverrides.runZoned(
+          () async {
+            const update = 'update';
+            const time = '(0.1s)';
+            final progress = Progress('message', stdout, stderr);
+            await Future<void>.delayed(const Duration(milliseconds: 100));
+            progress.update(update);
+            await Future<void>.delayed(const Duration(milliseconds: 100));
+            verify(
+              () {
+                stdout.write(
+                  '''${lightGreen.wrap('\b${'\b' * (update.length + time.length + 4)}⠹')} $update ${darkGray.wrap(time)}...''',
```

Comment:
This test is failing here. I am guessing that I'm not counting the `/b` or missing some "invisible" character. Would someone be able to help me understand why it might be failing?

Reply:
(Actually never mind.) I did not account the white space between the update message and the elapse time (`4` -> `5`)